docroot/modules/custom/mayflower/src/Render/SvgProcessor.php

```diff
@@ -25,27 +33,55 @@
  * placeholder will be swapped with an SVG use reference, embedding the SVG once
  * and using it everywhere.
  */
-class SvgProcessor implements AttachmentsResponseProcessorInterface {
+class SvgProcessor extends HtmlResponseAttachmentsProcessor {
 
   /**
-   * The decorated attachment processor.
+   * The HTML response attachments processor service.
    *
    * @var \Drupal\Core\Render\AttachmentsResponseProcessorInterface
    */
-  private $inner;
+  protected AttachmentsResponseProcessorInterface $htmlResponseAttachmentsProcessor;
 
   /**
-   * Constructor.
+   * Constructs a SvgProcessor object.
+   *
+   * @param \Drupal\Core\Render\AttachmentsResponseProcessorInterface $html_response_attachments_processor
+   *   The HTML response attachments processor service.
+   * @param \Drupal\Core\Asset\AssetResolverInterface $asset_resolver
+   *   An asset resolver.
+   * @param \Drupal\Core\Config\ConfigFactoryInterface $config_factory
+   *   A config factory for retrieving required config objects.
+   * @param \Drupal\Core\Asset\AssetCollectionRendererInterface $css_collection_renderer
+   *   The CSS asset collection renderer.
+   * @param \Drupal\Core\Asset\AssetCollectionRendererInterface $js_collection_renderer
+   *   The JS asset collection renderer.
+   * @param \Symfony\Component\HttpFoundation\RequestStack $request_stack
+   *   The request stack.
+   * @param \Drupal\Core\Render\RendererInterface $renderer
+   *   The renderer.
+   * @param \Drupal\Core\Extension\ModuleHandlerInterface $module_handler
+   *   The module handler service.
    */
-  public function __construct(AttachmentsResponseProcessorInterface $inner) {
-    $this->inner = $inner;
+  public function __construct(AttachmentsResponseProcessorInterface $html_response_attachments_processor, AssetResolverInterface $asset_resolver, ConfigFactoryInterface $config_factory, AssetCollectionRendererInterface $css_collection_renderer, AssetCollectionRendererInterface $js_collection_renderer, RequestStack $request_stack, RendererInterface $renderer, ModuleHandlerInterface $module_handler) {
+    $this->htmlResponseAttachmentsProcessor = $html_response_attachments_processor;
+    parent::__construct($asset_resolver, $config_factory, $css_collection_renderer, $js_collection_renderer, $request_stack, $renderer, $module_handler);
   }
 
   /**
    * {@inheritdoc}
    */
   public function processAttachments(AttachmentsInterface $response) {
     if ($response instanceof HtmlResponse) {
+
+      // (Note this is copied verbatim from
+      // \Drupal\Core\Render\HtmlResponseAttachmentsProcessor::processAttachments)
+      try {
+        $response = $this->renderPlaceholders($response);
+      }
+      catch (EnforcedResponseException $e) {
+        return $e->getResponse();
+      }
+
```

Comment:
Have you tried just adding these lines and not the rest of the changes? Does this fix the bug? I'm not sure how to reproduce the bug locally. As far as I can tell this is the only change that matters. The rest just reduces clarity as per my comment above. I see that BigPipe does it this way but occasionally core misses the mark.